cli/common/utils.go

```diff
@@ -582,7 +586,9 @@ func ContainsUpperSymbols(src string) bool {
 
 const (
 	appNameSpecifiedError = "Application name is specified. " +
-		"Please, specify instance name(s)"
-	instanceIDSpecified = `[APP_NAME].INSTANCE_NAME is specified. ` +
-		"Please, specify instance name(s)"
+		"You may have made a misprint. " +
```

Comment:
I think the "typo" word is more widely used in such cases

Reply:
Done